documentation/staging/content/samples/domains/model-in-image/prerequisites.md

```diff
@@ -138,7 +138,7 @@ weight: 1
    e. Later in this sample, when you run WebLogic Image Tool commands, the tool will use the image as a base image for creating model images. Specifically, the tool will implicitly call `docker pull` for one of the above licensed images as specified in the tool's command line using the `--fromImage` parameter. For `JRF`, this sample specifies `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4`, and for `WLS`, the sample specifies `container-registry.oracle.com/middleware/weblogic:12.2.1.4`.
 
      {{% notice info %}}
-   If you prefer, you can create your own base image and then substitute this image name in the WebLogic Image Tool `--fromImage` parameter throughout this sample. For example, you may wish to start with a base image that has patches applied. See [Preparing a Base Image]({{< relref "/userguide/base-images/_index.md" >}}).
+   If you prefer, you can create your own base image and then substitute this image name in the WebLogic Image Tool `--fromImage` parameter throughout this sample. For example, you may wish to start with a base image that has patches applied. See [Create a custom image with patches applied]({{< relref "/userguide/base-images/custom-images#create-a-custom-image-with-patches-applied" >}}).
```

Comment:
info --> warning

new wording can be something like:

The example base images are GA images that are suitable for demonstration purposes, but are not acceptable for production use. If you prefer, you can obtain a production ready image with the latest patches from OCR [link to OCR guidance], or [Create a custom image with patches applied]...

(Adjust above suggested wording to match the Warning wording in the image doc.)

Reply:
Done.

Reply:
Thanks. There is an additional issue. This might be an old problem, but the indentation is off -- the warning causes the following numbered bullet to start at the number "1" instead of "8" when displayed in hugo.

Reply:
I'll fix it.